lib/config.coffee

```diff
@@ -0,0 +1,13 @@
+module.exports =
+
+  apply: () ->
+
+    body = document.querySelector('html')
+
+    setLayoutMode = (layoutMode) ->
+      body.setAttribute('theme-one-dark-ui-layoutmode', layoutMode)
```

Comment:
You could maybe do `layoutMode.toLowerCase()` here just so you don't have to deal with title-cased attributes (which is a bit unusual).